README.rst

```diff
@@ -73,7 +73,7 @@ Fetch pubkey key for a server::
 
     curl http://localhost:8090/_matrix/identity/api/v1/pubkey/ed25519:0
 
-Internal bind api
+Internal bind and unbind api
 -----------------
 
 It is possible to enable an internal API which allows identifiers to be bound
```

Comment:
"It is possible to enable an internal API which allows for binding and unbinding between identifiers and matrix IDs without any validation."